### PR TITLE
Instructions and changes for running test with MySQL in Docker.

### PR DESCRIPTION
**Pull request by justinlittman**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -22,10 +22,14 @@ default: &default
 development:
   <<: *default
   database: sulbib_development
+  host: 127.0.0.1
```
Comment (jcoyne) [inline config/database.yml]: can we put these in default?

File: README.md [hunk 2]
```diff
@@ -40,6 +40,16 @@ bundle exec rake db:create
 bundle exec rake db:migrate
 ```
 
+Alternatively, you can use Docker:
+```
+docker run --rm -e MYSQL_ALLOW_EMPTY_PASSWORD=true -p 3306:3306 -d mysql:8
+```
+and then:
+```
+bundle exec rake db:create RAILS_ENV=test
```
Comment (jcoyne) [inline README.md]: The modern way to do this is `RAILS_ENV=test ./bin/rails db:create`

Comment (jcoyne) [reply README.md]: but, we aren't using rails 5 yet. Sigh.

File: config/database.yml (hunk 1 above)
Comment (justinlittman) [reply config/database.yml]: Yes. I had thought default was using unix sockets, but mistaken.

**Pull request merged**